### DISCUSSION TITLE
how to save object with original extension without using xl.meta ?

### DISCUSSION THREAD

**Discussion by nandarusfikri** [minio/minio#17017] [Q&A] [answered]
how to save object with original extension without using xl.meta ? 

By default .minio.sys uses xl.meta including config.

I want the objects stored in the bucket to use the original extension such as image.png files.

**Comment by klauspost** [comment]
The storage method is not configurable. As you may be able to tell `xl.meta` doesn't just contain the data.

You should not access the backend directly anyway.

**Reply by nandarusfikri** [reply]
previously used the minio version
Version: 2019-03-13T21:59:47Z
Release-Tag: RELEASE.2019-03-13T21-59-47Z

objects are saved with the original extension such as .png 

can the latest version save without xl.meta ?

**Reply by klauspost** [reply, marked as answer]
No.
